src/ledgers/five-bells-ledger.js

```diff
@@ -172,6 +179,96 @@ class FiveBellsLedger extends EventEmitter {
     return fulfillmentRes.body
   }
 
+  * _handleNotification (fiveBellsTransfer, relatedResources) {
+    if (fiveBellsTransfer.ledger !== this.id) {
+      throw new Error('Transfer was received by the wrong plugin (plugin: ' +
+        this.id + ', transfer: ' + fiveBellsTransfer.ledger + ')')
+    }
+
+    let handled = false
+    for (let credit of fiveBellsTransfer.credits) {
+      if (credit.account === this.credentials.account_uri) {
+        handled = true
+
+        const transfer = lodash.omitBy({
+          id: fiveBellsTransfer.id.substring(fiveBellsTransfer.id.length - 36),
+          direction: 'incoming',
+          ledger: this.id,
+          // TODO: What if there are multiple debits?
```

Comment:
Yeah, ILP would no longer used multi-crebit. I still think it's a good feature for a ledger to have and it could still be useful for us, so I don't see a strong reason to remove it, but we can chat about it.

It shouldn't be part of this PR, because it would be a breaking change. (And a fairly major one at that.)